minutes/meeting-202001019-4.md

```diff
@@ -0,0 +1,129 @@
+# Minutes of Meeting 4, 2020-10-17, 16:00–18:30 UTC
+
+Attendees
+
+Present
+liang, simon, Vlad, cjchapman, mehmetoguzderin, alerque, JohnHudson, dsinger, dscorbett, jfkthame, nmccully, Norbert
+
+* Chair: liang
```

Comment:
```suggestion
* Chair: Liang Hai
```

Either use names or IRC nicks, not both side by side in the same section. Maybe names through the attendance section, nicks for transcript?